Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -897,6 +905,8 @@ const TextInput = createReactClass({
           onBlur={this._onBlur}
           onChange={this._onChange}
           text={this._getText()}
+          isTabStop={windowsTabFocusable}
+          tabIndex={tabIndex}          
```

Comment:
>           [](start = 29, length = 10)

remove trailing whitespace

Reply:
hmm, didn't find this one...

Reply:
hmm, didn't find this one...

---
In reply to: [156557792](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156557792) [](ancestors = 156557792)